topics/01_General-and-Getting-Started/~Getting_Started.md

```diff
@@ -8,25 +8,237 @@
+        { "CountryName": "Brazil", "1995": 161, "2005": 186, "2015": 204, "2025": 218 }
+    ];
+
+    // Grid
+    $("#grid").igGrid({
+        width: "100%",
+        dataSource: populationData,
+        autoGenerateColumns: false,
+        columns: [{
+            key: "CountryName",
+            headerText: "Country",
+            width: "33.33%"
+        }, {
+            key: "2005",
+            headerText: "2005",
+            width: "33.33%"
+        }, {
+            key: "2015",
+            headerText: "2015",
+            width: "33.33%"
+        }],
+        features: [{
+            name: "Sorting",
+            columnSettings: [{
+                columnKey: "2015",
+                currentSortDirection: "descending"
+            }]
+        }]
+    });
+});
+```
+
+## <a id="page_designer"></a>Add an igGrid using Page Designer
+
+The Ignite UI Page Designer gives you a complete designer experience to configure any Ignite UI control by only using the mouse.
+To add `igGrid` to a page design area (on the left) in the toolbox (on the right) find "List & Pickers" section and drag and drop the Grid control. Then use the Property Editor to configure the grid. When ready just copy the resulting generated page.
+
+# <a id="custom_download"></a>Get just what you need
+
+The Ignite UI [Custom Download Page](https://www.igniteui.com/download) gives you the option to choose only the Ignite UI controls and features you use in your project and download optimized (minified and combined) JavaScript and CSS files for maximum page load performance.
+
+# <a id="cdn"></a>Using CDN Links
+
+Instead of hosting the Ignite UI script files into your project, you can just use Ignite UI CDN links. For Internet applications CDN usually serves files faster to the end users compared to host them on premise.
+
+Following are the Ignite UI Trial links. For more details checkout [Infragistics Content Delivery Network (CDN) for Ignite UI](deployment_guide_infragistics_content_delivery_network%28cdn%29.html) topic.
+
+```
+    <!-- Ignite UI Required Combined CSS Files (Trial) -->
+    <link href="http://cdn-na.infragistics.com/igniteui/latest/css/themes/infragistics/infragistics.theme.css" rel="stylesheet" />
+    <link href="http://cdn-na.infragistics.com/igniteui/latest/css/structure/infragistics.css" rel="stylesheet" />
+
+    <!-- JavaScript Library Dependencies -->
+    <script src="http://code.jquery.com/jquery-3.1.1.min.js"></script>
+    <script src="http://code.jquery.com/ui/1.12.1/jquery-ui.min.js"></script>
+
+    <!-- Ignite UI Required Combined JavaScript Files (Trial) -->
+    <script src="http://cdn-na.infragistics.com/igniteui/latest/js/infragistics.core.js"></script>
+    <script src="http://cdn-na.infragistics.com/igniteui/latest/js/infragistics.lob.js"></script>
+    <script src="http://cdn-na.infragistics.com/igniteui/latest/js/infragistics.dv.js"></script>
+```
+
```

Comment:
Can we also add a paragraph about ASP.NET MVC projects? 
We have samples and topics about that and the wrappers are bundled with the product so I think we must add them to the getting started topic.